### PR TITLE
Cg cli

### PR DESCRIPTION
**Pull request by mayabrandi**
CLI for loading status-db data. For now assuming json. Not a file but a string if that makes sense. This is so that it can be piped as output from cg. Format is json until we have decided something else.

### CODE REVIEW

File: vogue/commands/load/status_db.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import logging
+import click
+import json
+from vogue.tools.cli_utils import json_read
```
Comment (hassanfa) [inline vogue/commands/load/status_db.py]: Remove this line, we're not using it anymore.

File: vogue/commands/load/status_db.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+from vogue.tools.cli_utils import check_file
```
Comment (hassanfa) [inline vogue/commands/load/status_db.py]: same here. Remove this line, we're not using it anymore.

File: vogue/commands/load/status_db.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+@click.pass_context
+
+def status_db(context, db_dump, db_name):
+    """Reads status-db json dump"""
+
+    LOG.info("Reading db json dump.")
+    try:
+        db_dict = json.loads(db_dump)
+    except FileNotFoundError as e:
```
Comment (hassanfa) [inline vogue/commands/load/status_db.py]: Probably we should remove FileNotFoundError here. And add a log.error or something like that.

File: vogue/commands/load/status_db.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        context.abort()
+
+    db_dict = json.loads(db_dump)
```
Comment (hassanfa) [inline vogue/commands/load/status_db.py]: There is no need for this. And the next 3 lines. If json.loads fails, it'll throw exception anyway.

File: vogue/commands/load/status_db.py (hunk 1 above)
Comment (mayabrandi) [reply vogue/commands/load/status_db.py]: 👍

File: vogue/commands/load/status_db.py (hunk 3 above)
Comment (mayabrandi) [reply vogue/commands/load/status_db.py]: 👍

**Comment by ingkebil** [comment]
Hi guys,

some remarks:
- don't name the cli according to the tool you're importing from, but rather to the information you're about to set. The tool you're getting information from might change, get renamed, or replaced.
- name the command according to an action: build, set (singular values), transfer (complex data), ...
- if needed: set it to smaller increments, so you are able to set it manually.

I would rather go for:

```
vogue set sequenced-date $sample $date
vogue set received-date $sample $date
...
```
or
```
vogue transfer sequence-dates <(cg get samples --sequenced-after --json)
vogue transfer received-dates <(cg get samples --received-after --json)
...
```

Where `cg` spit out the json vogue needs.

What do you think?

**Pull request merged**